common/controllers/StorageController.ts

```diff
@@ -1,8 +1,10 @@
-import FileMetadata = firebase.storage.UploadMetadata;
+import firebase from 'firebase';
 import Firebase from 'common/services/firebase';
 import { createLogger } from 'common/logger';
 import Lazy from 'common/utils/lazy';
 
+type FileMetadata = firebase.storage.UploadMetadata;
+
```

Comment:
what's going on with this change here?

Reply:
I think that's a change from the export-functions. I've compared export-functions vs mock-backend, and it's not a change from the mock-backend branch.

Reply:
You can confirm it with the compare branch tool from `GitLens` extension in VSCode.